src/neomaril_codex/model.py

```diff
@@ -1030,13 +1043,13 @@ def __upload_model(self, model_name:str, model_reference:str, source_file:str,
             ("requirements", ("requirements.txt", open(requirements_file, 'rb')))
         ]
 
+        if schema:
+            upload_data.append(("schema", ("schema.json", parse_dict_or_file(schema))))
+        else:
+            raise InputError("Schema file is mandatory for Sync models")
```

Comment:
Schema field is now mandatory for Sync and Async models.
But for Async models it could be: 'json', 'csv', 'parquet', 'txt', 'xls', 'xlsx'
For Sync models it continues to be just 'json'

Reply:
@gkardoso I think it is important once now it is mandatory and can broke if user don't insert it.